Makefile

```diff
@@ -10,3 +10,4 @@ build-local: ## Builds a local executable of the project via "go build"
 
 start-local: build-local ## Builds and starts a local version of the program
 	@(cd server && ./main)
+	@(cd client && yarn start)
```

Comment:
Update the main "make start" script as well

Reply:
What should I update it to?

Reply:
Oh my B never mind, make start runs `docker-compose up` which also encompasses the frontend.